docs(jcloud): adapt to new api (#5385)
jina-ai/jina: docs/concepts/jcloud/yaml-spec.md

````{hint}

> By default, we attach an `efs` to all Executors in a Flow. The benefits of doing so is - it can grow dynamically, so you don't need to shrink/grow volumes as and when necessary.

Reviewer: ```suggestion
By default, we attach an `efs` to all Executors in a Flow. This lets the `efs` grow dynamically, so you don't need to shrink/grow volumes as and when necessary.
```

"it" in original text refers to "efs", right? Not Flow? Or Executor?

````{hint}

> By default, we attach an `efs` to all Executors in a Flow. This lets the `efs` grow dynamically, so you don't need to shrink/grow volumes as and when necessary.

Reviewer: ```suggestion
By default, we attach an `efs` to all Executors in a Flow. This lets the `efs` resize dynamically, so you don't need to shrink/grow volumes manually.
```

Reviewer 2: fixed!

### Set timeout

> By default, the JCloud gateway will close connections that have been idle for over 600 seconds. If you want longer a connection timeout threshold, you can change the `timeout` parameter under `gateway.jcloud`.

Reviewer: ```suggestion
By default, the JCloud gateway will close connections that have been idle for over 600 seconds. If you want a longer connection timeout threshold, change the `timeout` parameter under `gateway.jcloud`.
```

Reviewer 2: fixed

### Control gateway resources

> If you'd like to customize the gateway's CPU or memory, you can specify the `memory` and/or `cpu` argument under `gateway.jcloud.resources`:

Reviewer: ```suggestion
To customize the gateway's CPU or memory, specify the `memory` and/or `cpu` arguments under `gateway.jcloud.resources`:
```

Reviewer 2: 👍🏼 fixed

    uses: jinaai+docker://<username>/Encoder
```

## Expose Executors

> A Flow deployment without a gateway is often used for {ref}`external-executors`, which can be shared over different Flows. You can expose an Executor by setting `expose: true` (and un-expose the Gateway by setting `expose: false`):

Reviewer: ```suggestion
A Flow deployment without a gateway is often used for {ref}`external-executors`, which can be shared between different Flows. You can expose an Executor by setting `expose: true` (and un-expose the Gateway by setting `expose: false`):
```

Reviewer 2: 👍🏼 fixed

## Expose Executors

> A Flow deployment without a gateway is often used for {ref}`external-executors`, which can be shared between different Flows. You can expose an Executor by setting `expose: true` (and un-expose the Gateway by setting `expose: false`):

Reviewer: ```suggestion
A Flow deployment without a Gateway is often used for {ref}`external-executors`, which can be shared between different Flows. You can expose an Executor by setting `expose: true` (and un-expose the Gateway by setting `expose: false`):
```

Reviewer 2: fixed!
